draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1846,19 +1848,19 @@ Alternatively, the associated data can be fed into a collision-resistant KDF, su
 
```

Comment:
> The selected hash function must ensure a minimum of 128-bit preimage resistance.

Is this definitely correct? I've brought this up several times because surely there's an attack based on collision resistance (a collision = the same associated data)? The commitment papers always discuss collision resistance.

I think this should be changed to say `128-bit collision resistance`.

Reply:
My reasoning with preimage resistance was that it makes it impossible to set the AES words of the state to specific values. A hash function with a large output but no preimage resistance would be useless to prevent the attack.

But obviously, if the hash function is not collision resistant, different ADs would produce the same state.

So, we need both.

Reply:
Collision resistance added.